src/commonmark/en/content/user/managing-dashboards.md

```diff
@@ -202,16 +202,40 @@ a dashboard only applies to you, not other users.
 
 ### Filtering a dashboard
 
-To filter the dashboard by organisation unit, click on the **FILTER**
-button to the right of the dashboard title. Select the organisation
-units you want to apply to the dashboard. This only affects you, no
-other users. The only dashboard items that are impacted by the
-organisation unit filter are visualization items (pivot table, chart,
-map) that have their organisation unit configured to "User org unit". If
-you reload the dashboard, the filter will be cleared.
+Multiple filters can be applied to a dashboard for changing the data 
+displayed in the various dashboard items.
+The filters are applied to each dashboard item in the same way:
+each added filter overrides the original value for that dimension in
+the original analytical object.
+It is possible to filter on Organisation Units, Periods and other 
+dynamic dimensions depending on the DHIS2 instance.
+
+![](resources/images/dashboard/dashboard-filters.png)
+
+To add a filter, click on the **Add Filter** button and choose the
+dimension, a dialog opens where the filter selection can be made.
```

Comment:
I would put the text "To add a filter, click on the **Add Filter** button and choose the dimension" before the first image, and the text "Click on **Confirm** in the dialog to apply the filter to the current dashboard." after the second image so that there is a logical flow of text and images.

You could also add captions to the images, by adding text in the square brackets:
```
![Adding a filter](resources/images/dashboard/dashboard-filters.png)
![Org Unit filter selection](resources/images/dashboard/dashboard-orgunit-filter-dialog.png)
```
might make sense for these images.